diff --git a/node_modules/ses/dist/ses.cjs.js b/node_modules/ses/dist/ses.cjs.js
index ddf236d..af910ac 100644
--- a/node_modules/ses/dist/ses.cjs.js
+++ b/node_modules/ses/dist/ses.cjs.js
@@ -3403,6 +3403,7 @@ var dataPropertiesToRepair = {
         constructor: t$1, // set by "fast-json-patch"
         message: t$1,
         name: t$1, // set by "precond"
+	toString: t$1, // set by "bluebird"
       },
     },
 
diff --git a/node_modules/ses/dist/ses.cjs.js~ b/node_modules/ses/dist/ses.cjs.js~
new file mode 100644
index 0000000..ddf236d
