diff --git a/node_modules/stream-http/node_modules/readable-stream/errors-browser.js b/node_modules/stream-http/node_modules/readable-stream/errors-browser.js
index fb8e73e..e986f78 100644
--- a/node_modules/stream-http/node_modules/readable-stream/errors-browser.js
+++ b/node_modules/stream-http/node_modules/readable-stream/errors-browser.js
@@ -1,6 +1,10 @@
 'use strict';
 
-function _inheritsLoose(subClass, superClass) { subClass.prototype = Object.create(superClass.prototype); subClass.prototype.constructor = subClass; subClass.__proto__ = superClass; }
+function _inheritsLoose(subClass, superClass) { subClass.prototype = Object.create(superClass.prototype); Object.defineProperty(subClass.prototype, 'constructor', {
+  value: subClass,
+  enumerable: false,
+  writeable: true,
+}); subClass.__proto__ = superClass; }
 
 var codes = {};
 
@@ -29,7 +33,7 @@ function createErrorType(code, message, Base) {
     return NodeError;
   }(Base);
 
-  NodeError.prototype.name = Base.name;
+  Object.defineProperty(NodeError.prototype, 'name', { value:  Base.name });
   NodeError.prototype.code = code;
   codes[code] = NodeError;
 } // https://github.com/nodejs/node/blob/v10.8.0/lib/internal/errors.js
